.github/CODEOWNERS

```diff
@@ -72,13 +79,16 @@ iphone/Maps/LocalizedStrings/zh-Hans.lproj/ @organicmaps/translations-zh-Hans
 iphone/Maps/LocalizedStrings/zh-Hant.lproj/ @organicmaps/translations-zh-Hant
 android/src/fdroid/play/listings/ar/ @organicmaps/translations-ar
 android/src/fdroid/play/listings/be/ @organicmaps/translations-be
+android/src/fdroid/play/listings/bn-BD/ @organicmaps/translations-bn
 android/src/fdroid/play/listings/ca/ @organicmaps/translations-ca
 android/src/fdroid/play/listings/cs-CZ/ @organicmaps/translations-cs
+android/src/fdroid/play/listings/de-DE/ @organicmaps/translations-de
 android/src/fdroid/play/listings/eu-ES/ @organicmaps/translations-eu
 android/src/fdroid/play/listings/fr-FR/ @organicmaps/translations-fr
 android/src/fdroid/play/listings/it-IT/ @organicmaps/translations-it
+android/src/fdroid/play/listings/id/ @organicmaps/translations-id
 android/src/fdroid/play/listings/mr-IN/ @organicmaps/translations-mr
-android/src/fdroid/play/listings/pt-BR/ @organicmaps/translations-pt
+android/src/fdroid/play/listings/pt-BR/ @organicmaps/translations-pt-BR
```

Comment:
The same questions here. Do we have enough people to have separate teams?

Reply:
ditto ^^^